Pod/Classes/SKYDataSerialization.h

```diff
@@ -32,6 +32,7 @@ NSString *localFunctionName(NSString *remoteFunctionName);
 
 @interface SKYDataSerialization : NSObject
 
++ (NSDateFormatter *)dateFormatter;
```

Comment:
This means other classes will be able to change the format of `NSDateFormatter`.